wgsl/index.bs

```diff
@@ -12947,6 +12947,21 @@ The internal layout rules are described in [[#internal-value-layout]].
       <td>[=Component-wise=] reinterpretation of bits as `T`.<br>
       The result is the reintepretation of bits in `e` as a `vecN<T>` value.
 </table>
+<table class='data builtin'>
+  <tr><td style="width:10%">Overload
+      <td>
+        <xmp highlight=wgsl>@const @must_use fn bitcast<u32>(e : AbstractInt) -> T
+
+                            @const @must_use fn bitcast<vecN<u32>>(e : vecN<AbstractInt>) -> T</xmp>
+  <tr><td>Parameterization
+      <td>
+  <tr><td>Description
+      <td>Reinterpretation of AbstractInt as u32.
```

Comment:
This phrasing presupposes AbstractInt has a bit pattern.

Better to copy the wording from u32(AbstractInt):   `The same as u32(e)`.

> The identity operation if  e can be represented in [u32](https://gpuweb.github.io/gpuweb/wgsl/#u32), otherwise it produces a [shader-creation error](https://gpuweb.github.io/gpuweb/wgsl/#shader-creation-error).

> This is the same as `u32(e)`.